templates/_schemas/definitions/listItem.twig

```diff
@@ -2,8 +2,9 @@
   
   {% set schema = schema ?? craft.schema.ListItem %}
   {% set schema = schema
```

Comment:
This should also inherit from `Thing` just like ItemList. https://schema.org/ListItem

Reply:
Okay.

Reply:
Don't you have to call the `thing` macro somewhere? 🤔

Reply:
Sorry, fixed.